doc/uniqueIDGen.md

```diff
@@ -16,38 +16,114 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-# How to use UniqueIDGenSvc
 
-A Service to generate unique identifiers can be used with the following inputs:
-- Event number, Run number and Algorithm name
-- Seed number: set in the options python file
+# Seeding algorithm PRNG
 
-To set a seed for the UniqueIDGenSvc, add the following to an options file:
+Correctly seeding pseudo-random number generators (PRNG) is important to ensuring reproducible and good-quality randomness. A common pattern is to reinitialize each algorithm’s PRNG with a unique seed for each processed event.
+
+## UniqueIDGenSvc service
+
+The `UniqueIDGenSvc` Gaudi service can generate unique `size_t` values that can be used as PRNG seeds, based on an event number, run number, and algorithm.
+
+The service can be included in a steering file by:
 
 ```python
 from Configurables import UniqueIDGenSvc
-UniqueIDGenSvc().Seed = 987654321
+uidgen_svc = UniqueIDGenSvc()
+```
+
+The service can be configured with a central seed from which the unique values will be derived. This seed can be configured with the `Seed` property:
+
+::::{tab-set}
+:::{tab-item} Python
+```python
+uidgen_svc.Seed = 987654321
+```
+:::
+:::{tab-item} CLI
```

Comment:
Is it possible to directly link to the CLI part of this tabbing?

Reply:
Thanks for investigating! I think I would propose to have this linkable somehow?